src/views/CartView.vue

```diff
@@ -33,6 +38,10 @@
                 to="/order"
                 tag="button"
               >{{$t('lifelines-webshop-order-btn-label')}}</router-link>
+
+              <button v-if="hasManagerRole" class="btn btn-secondary ml-2" @click="toggleAllVariables">
```

Comment:
2 buttons ?

Reply:
I figured that the button should be aligned with the save/order buttons, but those are only rendered when there are selected variables; otherwise the other block is rendered.